images/builders/mlflow/Dockerfile

```diff
@@ -48,6 +48,6 @@ ENV FUSEML_INTEL_OPTIMIZED false
 ENV FUSEML_BASE_IMAGE ""
 ENV FUSEML_VERBOSE false
 # Set to false to reduce memory usage with larger images and avoid OOM problems
```

Comment:
shouldn't you change the comment now to explain the reasons for changing to true?

Reply:
done